contracts/Vault.sol

```diff
@@ -331,15 +333,32 @@ contract Vault is
     {
         require(address(strategy) != address(0), "Vault: strategy is not set");
 
-        uint256 _investable = investableAmount();
+        (uint256 maxInvestableAmount, uint256 alreadyInvested) = investState();
+
+        require(
+            maxInvestableAmount > alreadyInvested
+                ? maxInvestableAmount - alreadyInvested != 0
+                : alreadyInvested - maxInvestableAmount != 0,
```

Comment:
Maybe I'm confused, but isn't this the same as 

```
require(maxInvestableAmount != alreadyInvested, "Vault: nothing to invest or disinvest");
```

?

Reply:
yes :rofl: rofl: rofl: 

adjusted, sorry about that, I blame sleep deprivation